README.md

```diff
@@ -320,9 +320,6 @@ init_trainer()
 # Launch DDP on 4 GPUs'
 setup_device(multi_gpu=MultiGPUMode.DISTRIBUTED_DATA_PARALLEL, num_gpus=4)
 
```

Comment:
Still need to instantiate the trainer when launched this way.

Reply:
changed in dcde517e71600e4183edb385da5d897b30830e4c